packages/tokens/src/create-theme.cjs

```diff
@@ -245,6 +245,17 @@ const getCoreTokensByType = (tokenType, cssVariableScope) => {
       })${description ? ` /* ${description} */` : ''}`;
     }
   });
+  return result;
+};
+
+const getZIndices = () => {
```

Comment:
@mariohamann @coraliefeil Do we need here some logic for which "settings/components" are open for z-index?